joomla/joomla-cms
# Composer update

### Summary of Changes

composer update --no-dev --ignore-platform-reqs

  - Removing symfony/polyfill-util (v1.19.0)
  - Updating joomla/archive (1.1.8 => 1.1.10): Downloading (100%)
  - Removing symfony/polyfill-php55 (v1.19.0)
  - Installing symfony/polyfill-php55 (v1.20.0)
  - Removing symfony/polyfill-php56 (v1.19.0)
  - Installing symfony/polyfill-php56 (v1.20.0)
  - Removing symfony/polyfill-php71 (v1.19.0)
  - Installing symfony/polyfill-php71 (v1.20.0)
  - Updating symfony/polyfill-php73 (v1.19.0 => v1.23.0): Loading from cache
  - Updating google/recaptcha (1.1.2 => 1.2.4): Loading from cache
  - Updating psr/container (1.0.0 => 1.1.1): Loading from cache
  - Updating symfony/polyfill-ctype (v1.19.0 => v1.23.0): Loading from cache
  - Updating psr/log (1.1.3 => 1.1.4): Loading from cache
  - Removing leafo/lessphp (dev-joomla3-php8)
  - Installing leafo/lessphp (dev-joomla3-php8 a149187): Cloning a149187adb from cache

### Testing Instructions

Make sure the following packages still work:
- recaptcha
- lessphp (generation of less files from within the template manager)
- joomla/archive (upload + install an extension)

### Actual result BEFORE applying this Pull Request

non up-to-date packages

### Expected result AFTER applying this Pull Request

up-to-date packages

### Documentation Changes Required

none
